### ISSUE TITLE
User story - Job search

### ISSUE THREAD

**Issue by stephendawsondev** [stephendawsondev/StackPortfolio#35]
### If applicable, add a related Epic.

Epic: #28 , #42 

### User Story

As a **user**, I can **search jobs** so that I can **find the most relevant job to me**.

### Acceptance Criteria

- The user should be at least able to search by the position
- Search results should be display in carts on a search page

### Please ensure you've completed these tasks before marking the issue as done.

- [x] Reviewed the user story for clarity and completeness.
- [x] Identified and linked related issues or epics.
- [x] Created or updated the necessary documentation.
- [ ] Written unit and/or integration tests for functionality.
- [x] All code changes have been manually tested to ensure they meet the acceptance criteria.
- [x] Updated the issue with any relevant notes or feedback (to date).
- [x] Merged into the main branch and deployed to production (if applicable).

### Please add any further tasks that need to be completed before the issue can be marked as done.

- [x] Create a search page

**Comment by stephendawsondev** [comment]
This was completed in #96
